packages/catalog-client/src/CatalogClient.ts

```diff
@@ -468,51 +416,21 @@ export class CatalogClient implements CatalogApi {
   // Private methods
   //
 
-  private async requestIgnored(
-    method: string,
-    path: string,
-    options?: CatalogRequestOptions,
-  ): Promise<void> {
-    const url = `${await this.discoveryApi.getBaseUrl('catalog')}${path}`;
-    const headers: Record<string, string> = options?.token
-      ? { Authorization: `Bearer ${options.token}` }
-      : {};
-    const response = await this.fetchApi.fetch(url, { method, headers });
-
+  private async requestIgnored(response: Response): Promise<void> {
```

Comment:
I think we could tweak the naming of these now that they're not executing the request and just handling the response. Feeling we can leave that for a followup though